### DISCUSSION TITLE
[M-Z] (15 Gennaio 2018) Esercizio 2

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#424] [Esami & Esoneri M-Z]
## Traccia

<img width="929" alt="image" src="https://github.com/user-attachments/assets/684df15b-03f4-4295-a856-d2b6bf9442f5">

> Questo esercizio è stato proposto **anche** come **Esercizio 2** della traccia di esame **del 22 gennaio 2019**

**Comment by rimaout** [comment]
<img width="600" alt="image" src="https://github.com/user-attachments/assets/d4a5e12b-892d-4a9c-9aad-37dae8273db3">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [*20190122.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20190122.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/2b681519-b9b0-49f9-a783-4556af7333ff)

\* _Questo esercizio era in comune rispetto alla traccia del 22 gennaio 2019, gli esercizi 7, 8 e 9 sono specifici delle due rispettive tracce, per quelli vedi [20180115.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20180115.pdf)_
